### PR TITLE
feat(LINGUIST-436): AWS Polly support for chatbot voices

### PR DESCRIPTION
**Pull request by selimcanglr**

### CODE REVIEW

File: %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver [hunk 1]
```diff
@@ -0,0 +1 @@
+1216.2402.13012.0
```
Comment (can-aslan) [inline %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver:1]: Is this supposed to be here?

Comment (selimcanglr) [reply %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver:1]: No 😆

Comment (selimcanglr) [reply %ProgramData%/Microsoft/Windows/UUS/State/_active.uusver]: Removed it

**Review by can-aslan — APPROVED** [review]
🔉 🔉 🔉 🔉 🔉

**Pull request merged**